images/win/scripts/Installers/Install-Go.ps1

```diff
@@ -13,17 +13,20 @@ function Install-GoVersion
         [Switch]$addToDefaultPath
     )
 
+    $latestVersionObject = $refsJson | Where-Object { $_.ref -Match "refs/tags/go$goVersion[./d]*" }  | Select-Object -Last 1
+    $latestVersion = $latestVersionObject.ref -replace "\D+\D+[./d]*"
```

Comment:
nitpick: $latestVersionObject.ref.replace('refs/tags/go','') is simple than regex

Reply:
Done.